### PR TITLE
TNW Prema Building with Responsive Design - Milestones 1

### PR DESCRIPTION
**Pull request by verissimohenry**
Due to the fact I didn't get a partner, I was asked to fork from my last partner and submit on my own.
it's the milestone one of my project already started the milestone two

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,788 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href=Css/Css-reset.css />
+    <link rel="stylesheet" href="Css/Style.css" />
+    <title>TNW-PREMA</title>
+  </head>
+
+  <body>
+    <section id="nav-sec">
```
Comment (cyonii) [inline index.html]: You should use the `header` tag for this header.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,788 @@
...
+        </div>
+        <nav class="naa-1">
+          <div class="nav-logo" style="background-color: black">
+            <a
+              href="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646mp_/https:/thenextweb.com/"
+              ><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 66 16"><path fill="#FFF" d="M34 16h-6v-5.2c0-.7-.3-1.3-.7-1.8L25 6.7c-.5-.4-1.1-.7-1.8-.7H22v10h-6V0h8.7c1.3 0 2.6.5 3.5 1.5l4.3 4.3c1 .9 1.5 2.2 1.5 3.5V16zM60 0v5.2c0 .7-.3 1.3-.7 1.8L57 9.3c-.5.5-1.1.7-1.8.7h-2.5c.8-.9 1.3-2.1 1.3-3.3V0h-6v5.2c0 .7-.3 1.3-.7 1.8L45 9.3c-.5.5-1.1.7-1.8.7H42V0h-6v16h8.7c1.3 0 2.6-.5 3.5-1.5l2.8-2.8V16h5.7c1.3 0 2.6-.5 3.5-1.5l4.3-4.3c1-.9 1.5-2.2 1.5-3.5V0h-6zM14 0H0v6h4v10h6V6h4"/></svg></a>
+          </div>
+          <div class="hambutt">
```
Comment (cyonii) [inline index.html:32]: Your hamburger button should use an `a` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,788 @@
...
+            <div class="ham"></div>
+            <div class="ham"></div>
+            <div class="ham"></div>
+          </div>
+          <img
```
Comment (cyonii) [inline index.html]: Your search icon should be an `a` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,788 @@
...
+            >Calling young digital talent in the Netherlands – Be recognized in
+            the T500 →</a
+          >
+        </nav>
+      </nav>
+    </section>
+
+    <section class="main-sec">
+      <!--main tag-->
+      <section class="article3">
```
Comment (cyonii) [inline index.html]: You have nested `section` tags here, section elements should be used to group related content. If you just need to position or style elements, then you should probably use `div`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,788 @@
...
+        <article class="art-cont art-main">
+          <div class="art1 art123">
+            <a class="art-cover" href="#"></a>
+            <header class="arthead">
```
Comment (cyonii) [inline index.html]: The `header` tag should be used should be used only in the actual header of your document, please remove them in other places. You should use `h1` to `h6` to give your articles heading. Please do this for all occurrences. They article title should be clickable links.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,788 @@
...
+              </div>
+            </a></div>
+            <div>
+              <ul>
+                <li><a href="#">Telecommunications</a></li>
+                <li><a href="#">Expedition Growth Capital</a></li>
+              </ul>
+            </div>
+            <div>
+              <p>
```
Comment (cyonii) [inline index.html:269]: This text excerpt should a link, `a` tag

File: index.html [hunk 7]
```diff
@@ -0,0 +1,788 @@
...
+                </div>
+            </div>
+          <div class="cover-gradient"></div> 
+        </div>
+    </section> -->
+
+
+      <section class="ltst-fndg">
+        <a href="#"><h2>Latest funding rounds on index.co ></h2></a>
+        <ul>
```
Comment (cyonii) [inline index.html]: Your nav menu (See more, Get updates, etc.) should be grouped in a nav tag and each a have link, `a` tag, and should be separated by `|`

**Review by cyonii — CHANGES_REQUESTED** [review]
# Status: Changes Required ✍️

Hello @verissimohenry, you have shown a great effort for this project so far. I will like to highlight the following areas where you've shown excellence:
- Good job covering this milestone in your README doc
- Good use of media queries to 

## To Improve
You've shown a great effort so far, however, there are few defects you need to fix in order to get this project approved.
- [ ] Generally, you need to improve your use of semantic HTML, please check the inline comments and fix the issues pointed out.
- [ ] From tablet screen and up, the title of the first article should be bigger than the other two.
- [ ] The TWN logo should appear over the first article at the top left corner on a red overlay. Your versionOriginal
- [ ] Please take a look at this card cards and fix the alignments and the absent texts on mobile view
Your versionOriginal version


## Nice to Have

These suggestions are OPTIONAL, which means you do not have to implement them.
- [ ] [OPTIONAL] You should use `font-family: sans-serif` for better look.



#### You can find me on slack 👉 [CY Kalu](https://microverse-students.slack.com/team/U01494U96FR) if you need further assistance or clarification. I'm always available and will be glad to help.

File: .github/workflows/linters.yml [hunk 8]
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Comment (christianotieno) [inline .github/workflows/linters.yml:1]: ## General Requirements for this Microverse Project
- [ ] By now you should be comfortable with creating a more detailed, neat-looking and professional `Pull Request` when sending them in for review. I understand that you got to work on this project alone and as a result, there was a need for you to indicate this. I however believe the information ought to have been included in the `Additional Information` section of the `Ask for Review` form when requesting for a reive. Kindly note that you need to ensure you are always using meaningful names for your `Pull Requests` and add a summary of its content(essential, what got carried out in this project) just as described [here](https://microverse.zendesk.com/hc/en-us/articles/360055660814-Professional-GitHub-repository).

- [ ] While at it, the project also ought to have a much descriptive commit message. They ought to act as a tracking and guiding tool for changes in the project's codebase. As of now, it is hard to follow through with this. The `fixing linter errors` & `fix linter errors` were so rampant that one cant clearly tracks when a feature, fix to a bug or new changes got added to the project and at what point(commit). In your next round of changes ensure you use descriptive commit messages as described [here](https://microverse.zendesk.com/hc/en-us/articles/360055660814-Professional-GitHub-repository).

File: Css/Style.css [hunk 9]
```diff
@@ -0,0 +1,1360 @@
...
+}
+
+/* *************************Navigation bar 2************************ */
+.nav-logo-tablet {
+  display: none;
+}
+
+.toprow-2 [class^="na-"] {
+  display: none;
+}
```
Comment (christianotieno) [inline Css/Style.css:25]: ## Aim to stay `DRY`
- [ ] Class selectors on lines `#20` & `#24` seem to be sharing a rule.. Do you mind merging them so as to appear not to be repeating yourself?  There are far too many of these incidents on this file. For instance, classes `art1`, `art2` & `art3` have lots of shared rules in them, track them and find a way of bundling these rules in one place to appear neat and uses code as well. Do this for all such instances.

- This section requires that you ensure you are NOT making [common mistakes](https://speckyboy.com/good-bad-css-practices) in your CSS styles.

File: README.md [hunk 10]
```diff
@@ -1 +1,55 @@
...
+
+
+## Built With
+
+- HTML
+- CSS
+## Live Demo
+
+[The Next Web](https://verissimohenry.github.io/TNW-Prema/)
+
```
Comment (christianotieno) [inline README.md:17]: ## Improve the live link
- [ ] The live link to where the page is hosted appear to be broken. Do you mind fixing this?

File: README.md [hunk 11]
```diff
@@ -1 +1,55 @@
...
+👤 **Author1**
+# Abdulrahman Naser
+- GitHub: [@githubhandle](https://github.com/Abdona)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abdulrahman-nasser-2b7173131/)
+
+👤 **Author2**
+# Verissmo Henry
+- GitHub: [@githubhandle](https://github.com/verissimohenry)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/henry-verissimo-618906167/)
+
```
Comment (christianotieno) [inline README.md]: ## Improve the Author section
- [ ] I understand that you worked on this project alone. Would you mind only adding contributors that worked on this project in this section and leave out the rest? 
- [ ] While at it. I would like to suggest a neat way of showcasing the contributors to this project in this section so 👇🏾 This ensures you use fewer writings than necessary for fear of repeating yourself. For instance, I would replace the `Author1` with your actual name and replace the social placeholder name with your actual usernames.

```suggestion
## Authors
👤 **Verissmo Henry**
- GitHub: [@verissimohenry](https://github.com/verissimohenry)
- LinkedIn: [Henry Verissmo](https://www.linkedin.com/in/henry-verissimo-618906167/)

```

File: README.md [hunk 12]
```diff
@@ -1 +1,55 @@
...
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Abdona/Project_Positioning-Floating/pull/3#issue-576494545).
```
Comment (christianotieno) [inline README.md]: ## FIX this link
- [ ] The issues link is getting users redirected to a different repository other than the one intended. Kindly fix this.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,785 @@
+<!DOCTYPE html>
```
Comment (christianotieno) [inline index.html:1]: ## Improve the markup
- [ ] Kindly ensure you are using semantic HTML. You can confirm this by validating your markup. Please feel free to use this [tool](https://validator.w3.org/nu/#textarea) and work on the informing warnings.

File: index.html [hunk 14]
```diff
@@ -0,0 +1,785 @@
...
+                  <li>2 hour ago</li>
+                  <li>5</li>
+                </ul>
+              </div>
+            </header>
+          </div>
+        </article>
+      </section>
+      
+    <!--<div id="title1">
```
Comment (christianotieno) [inline index.html:139]: ## Remove old pieces of commented code.
- [ ] Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history just as explained [here](https://microverse.zendesk.com/hc/en-us/articles/360057489093-General-Requirements-for-Microverse-Projects).

File: index.html [hunk 15]
```diff
@@ -0,0 +1,785 @@
...
+              <div>
+                  <a class="special_text" href="#">Abhimanyu Ghoshal</a>
+                  <p class="special_text">· 8 hours ago</p>
+              </div>
+          </div>
+
+      </div>
+    </section>-->
+
+    <section class="ltst-fndg">
```
Comment (christianotieno) [inline index.html:237]: ## Improve the layout of the latest funding section.
### on a large screen.
- [ ] Kindly observe that the three main articles of the page seem to be occupying the entire width of the screen in large screen breakpoints. The rest sections' contents saved for the footer of the page seem to be covering lesser width fo the page and are all centred.  see the comparison below 👇🏾 

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/verissimohenry/TNW-Prema/mileston1-mainsection/index.html)|
|:--:|:--:|
|latest-funding|latest-funding-clone|

### On small screens 
- [ ] Ensure that all the elements in this section are all aligned in a column and are all covering the entire view width of the page. 

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/verissimohenry/TNW-Prema/mileston1-mainsection/index.html)|
|:--:|:--:|
|latest-funding-small-screen|latest-funding-small-screen-clone|

File: index.html [hunk 16]
```diff
@@ -0,0 +1,785 @@
...
+                Food delivery startup Wolt raises $530M after tripling revenues <span>on SiliconANGLE</span>
+              </p>
+            </div>
+          </div>
+        </div>
+    </div>
+  </section>
+</section>
+
+<!-- <section id="section4">
```
Comment (christianotieno) [inline index.html]: ## Remove old pieces of commented code.
- [ ] Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history just as explained [here](https://microverse.zendesk.com/hc/en-us/articles/360057489093-General-Requirements-for-Microverse-Projects).

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Chnages Required ♻️ 
Hello @verissimohenry 👋🏾 
The project is coming along just fine with the changes Kinly take note on the following changes before this project gets approved.

### Mandatory changes 
- [ ] Ensure there are no [common mistakes](https://speckyboy.com/good-bad-css-practices) in your CSS styles.
- [ ] Aim to use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) in this project.
- [ ] Fix the live link of the page in the README
- [ ] Improve the Pull Request message and commit messages
- [ ]  Fix the general layout of the page.
- [ ] Remove Commented out code.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
# Status: ♻️ CHANGES REQUIRED ♻️
_Great job guys so far 👏 💯 👍. However, there are few issues we need to address before we move on, please check the comments below._

- ### To Highlight
  - [x] The project is clean. 💯
  - [x] The project follows Git flow. 💯

- ### To Improve
  - [x] [MANDATORY] Please check out this point  "Improve the layout of the latest funding section : on a large screen." In the previous feedback and fix it.


_Please make all the required changes before you submit another code review. If you have any problems or concerns, you can reach out to me on Slack: Rukundo Eric 🚀 . I wish you all the very best 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/), 
Slack: Rukundo Eric**

**Review by mke2111 — CHANGES_REQUESTED** [review]
# **STATUS: CHANGES REQUIRED 🔴 ⚠️** 

## Hey @verissimohenry, 👋🏽 
 **Thank you for the good job you got going on and implementing a few of the requested changes.**

## REQUIRED CHANGES

- [x] You have endeavored to make a few of the required changes. Kindly fix all the requested changes. Read my inline comments below. ⚠️ 

**Kindly make all the requested changes and submit another CRR. 👍🏽** 

## Cheers!! 🍹 & Happy Coding.

# Reach me on slack for a chitchat 🆘  **Roy Mukuye**

File: index.html [hunk 17]
```diff
@@ -0,0 +1,772 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href=Css/Css-reset.css />
+  <link rel="stylesheet" href="Css/Style.css" />
+  <title>TNW-PREMA</title>
+</head>
+
+<body>
+  <section id="nav-sec">
+    <nav class="toprow-2">
```
Comment (mke2111) [inline index.html:15]: This should be a `header` tag.

Please read on how to use semantic HTML appropriately.

File: index.html [hunk 18]
```diff
@@ -0,0 +1,772 @@
...
+        <img class="srchicon" alt="search icon" src="Assets/Images/magnifying-glass.svg" />
+      </nav>
+      <nav class="na-4">
+        <a href="#">Calling young digital talent in the Netherlands – Be recognized in
+          the T500 →</a>
+      </nav>
+    </nav>
+  </section>
+
+  <section class="main-sec">
```
Comment (mke2111) [inline index.html]: Change this to the `main` element.

File: index.html [hunk 19]
```diff
@@ -0,0 +1,772 @@
...
+        <ul class="nav-ul">
+          <li><a href="#">LATEST</a></li>
+          <li><a href="#">HARD FORK</a></li>
+          <li><a href="#">PLUGGED</a></li>
+          <li><a href="#">README</a></li>
+          <li><a href="#">GROWTH QUARTERS</a></li>
+          <li><a href="#">SHIFT</a></li>
+          <li><a href="#">NEURAL</a></li>
+        </ul>
+        <img class="srchicon" alt="search icon" src="Assets/Images/magnifying-glass.svg" />
```
Comment (mke2111) [inline index.html]: The search icon should be enclosed in an `a` tag.

File: index.html [hunk 20]
```diff
@@ -0,0 +1,772 @@
...
+
+  <section class="main-sec">
+    <!--main tag-->
+    <section class="article3">
+      <article class="art-cont art-main">
+        <div class="art1 art123">
+          <a class="art-cover" href="#"></a>
+          <header class="arthead">
+            <div class="big-logo">
+              <img alt="" src="Assets/Images/big-logo.svg" />
```
Comment (mke2111) [inline index.html]: The  logo should be enclosed in an `a` tag.

File: index.html [hunk 21]
```diff
@@ -0,0 +1,772 @@
...
+            </div>
+            <div class="date">Tuesday — January 26, 2021</div>
+            <div class="opal">
+              <p>
+                Opal cuts off your apps' internet access to help you stop
```
Comment (mke2111) [inline index.html]: This title should be enclosed in both `a` and `hx` tags. Do the same thing for all the titles in the Highlight section

File: index.html [hunk 22]
```diff
@@ -0,0 +1,772 @@
...
+    </nav>
+  </section>
+
+  <section class="main-sec">
+    <!--main tag-->
+    <section class="article3">
+      <article class="art-cont art-main">
+        <div class="art1 art123">
+          <a class="art-cover" href="#"></a>
+          <header class="arthead">
```
Comment (mke2111) [inline index.html]: Remove this `header` tag. It is inside what is meant to be the `main` tag. The Proper HTML structure should be 

1. `header`
2. `main`
3. `footer`

Comment (mke2111) [reply index.html]: You can change it to a `div`

File: index.html (hunk 14 above)
Comment (mke2111) [reply index.html:139]: As requested previously, remove all commented out code from this file.

File: index.html (hunk 22 above)
Comment (verissimohenry) [reply index.html]: thank you for the corrections, I have made the changes.

File: index.html [hunk 23]
```diff
@@ -0,0 +1,772 @@
...
+      </a>
+      <div></div>
+      <nav class="more">
+        <a href="#" class="more">Add info</a>
+        <p class="hide">|</p>
+        <a id="border_LA" href="#" class="more">Get updates by email</a>
+        <p class="hide">|</p>
+        <a href="#" class="more">See more</a>
+      </nav>
+      <div class="founding-cnt">
```
Comment (mke2111) [inline index.html]: This should be an `article` tag. Same applies to all the other 3 remaining 'cards'.

File: index.html [hunk 24]
```diff
@@ -0,0 +1,772 @@
...
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="fndg-3">
+          <div class="fng-row">
+            <div class="fng-head">
+              <a href="#">
+                <img
+                  src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F1198933202947203078%2FyGxNiAdA.png&amp;signature=d0c4ef35a8e410806bed50513f59f79b"
```
Comment (mke2111) [inline index.html]: This logo image is broken, same with the next two articles' logos.

File: index.html [hunk 25]
```diff
@@ -0,0 +1,772 @@
...
+                  alt="#" />
+                <h4>SwissdeCode</h4>
```
Comment (mke2111) [inline index.html:282]: This title should be enclosed in an `a` tag. Do the same thing in the other articles as well

File: index.html [hunk 26]
```diff
@@ -0,0 +1,772 @@
...
+        <div>
+            <h4><a href="#"> Familio Private Backup Platform: Lifetime Subscription</a></h4>
+            <span class="original_price">$300 </span>
+            <span class="new_price">$39.99</span>
+        </div>
+    </div>
+</section> -->
+
+  <footer class="tnw-foot">
+    <div class="foot-icn">
```
Comment (mke2111) [inline index.html:708]: All social media icons should be enclosed in a `nav` tag. Each icon should be enclosed in an `a` tag.

File: index.html [hunk 27]
```diff
@@ -0,0 +1,772 @@
...
+        <path
+          d="M12 12.713l-11.985-9.713h23.971l-11.986 9.713zm-5.425-1.822l-6.575-5.329v12.501l6.575-7.172zm10.85 0l6.575 7.172v-12.501l-6.575 5.329zm-1.557 1.261l-3.868 3.135-3.868-3.135-8.11 8.848h23.956l-8.11-8.848z" />
+      </svg>
+      <svg class="icon" xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 24 24">
+        <path d="M14 18l10-7.088-10-6.912v3.042s-11.618 2.583-14 12.958c5.072-5.431 14-5.218 14-5.218v3.218z" />
+      </svg>
+    </div>
+    <div class="foot-lst">
+      <!--NAV tag-->
+      <ul>
```
Comment (mke2111) [inline index.html:733]: This should also be a `nav` tag

**Comment by mke2111** [comment]
## Kindly make all the requested changes to achieve mastery. 
Best of luck.

🍹 🕺🏽

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations team :tada: 

**This project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:**

**Comment(s)**
- On desktop screen sizes, you seem to be having more space to the left than on the right side. Kindly fix this against your next milestone.

Again, congratulations, you can now merge to master.
Good luck with your next task.

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__
